bin/internal/shared.bat

```diff
@@ -13,10 +13,12 @@ REM --------------------------------------------------------------------------
 
 SETLOCAL
 
-REM Ensure we are runnng on 64-bit windows (32-bit is not supported)
+REM Ensure we are running on 64-bit windows (32-bit is not supported)
```

Comment:
Why this is necessary isn't immediately obvious. Consider adding a quick comment to explain this

```suggestion
REM Ensure we are running on 64-bit Windows (32-bit is not supported).
REM If this is a 32-bit process emulated by WOW64,
REM PROCESSOR_ARCHITECTURE is the process architecture and
REM PROCESSOR_ARCHITEW6432 is the processor architecture.
```